snippets/csharp/VS_Snippets_ADO.NET/DataWorks DataTableLoad.IDataReader/CS/source.cs

```diff
@@ -32,14 +32,8 @@ static void Main()
         // by the particular incompatibility:
         table = GetIntegerTable();
         reader = new DataTableReader(GetStringTable());
-        try 
```

Comment:
This should be caught - with the right DB exception.

Reply:
If this message is for end-user, what is the idea we show "Out of memory" or "Object Null Reference"? If this is for developer, exception message will go to console output automatically. Removing catch also makes attached debugger stopped on the method that throws exception (for developer it should be more practical than to see only message in console?)

Reply:
If this message is for end-user, what is the idea we show "Out of memory" or "Object Null Reference"? If this is for developer, exception message will go to console output automatically. Removing catch also makes attached debugger stopped on the method that throws exception (for developer it should be more practical than to see only message in console?)